### PR TITLE
Fixed commas

### PR DESCRIPTION
**Pull request by ryanmaclean**

### CODE REVIEW

**Comment by monicasarbu** [comment]
@dedemorton can you please have a look at this PR? Thanks!

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ gathered by the Elastic [Beats](https://www.elastic.co/products/beats).
 Installation
 -------------
 
-To load the dashboards, execute the script pointing to the Elasticsearch HTTP
+To load the dashboards execute the script pointing to the Elasticsearch HTTP
```
Comment (dedemorton) [inline README.md:10]: The original sentence is grammatically correct. A comma usually follows introductory phrases of four or more words.

File: README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ URL:
         # Windows
         .\load.ps1 -url "http://localhost:9200"
 
-If you want to use HTTP authentication for Elasticsearch, you can specify the
+If you want to use HTTP authentication for Elasticsearch you can specify the
```
Comment (dedemorton) [inline README.md:19]: Same comment as above.

File: README.md [hunk 3]
```diff
@@ -52,11 +52,11 @@ script is described above.
 Note: Make sure you are using the latest Kibana version to create and download
 the dashboards.
 
-Then, you can create the dashboard together with the necessary
-visualizations and searches in Kibana. After the dashboard is ready, you can download
```
Comment (dedemorton) [inline README.md:56]: Same comment as above. The removal of the comma after "then" is correct.

File: README.md [hunk 4]
```diff
@@ -52,11 +52,11 @@ script is described above.
...
 the dashboards.
 
-Then, you can create the dashboard together with the necessary
-visualizations and searches in Kibana. After the dashboard is ready, you can download 
+Then you can create the dashboard together with the necessary
+visualizations and searches in Kibana. After the dashboard is ready you can download 
 all the dashboards using the `save/kibana_dump.py` script. 
 
-Before executing the save/kibana_dump.py script, make sure you have python and virtualenv
+Before executing the save/kibana_dump.py script make sure you have python and virtualenv
```
Comment (dedemorton) [inline README.md:59]: Same comment as above.

**Comment by dedemorton** [comment]
@ryanmaclean Thanks for your fix. Can you roll back the changes and update the PR? The changes that I haven't marked look OK, but the other changes are not consistent with Elastic style.

**Comment by ryanmaclean** [comment]
Definitely more pausing than I'm used to, but I can see where you're going with this. 
The comma after "Then," matches the "Finally," at the end of the doc. 

Generally I avoid them in small phrases, but if your style is to break them up, I get that too :) 

Will close the PR for now - I think there's only one change to make?

**Comment by dedemorton** [comment]
@ryanmaclean The Chicago Manual of Style is a good resource to consult if you have questions about specific rules.

**Pull request closed without merging**